libs/libc/misc/lib_cxx_initialize.c

```diff
@@ -75,13 +77,18 @@ extern uintptr_t _etext;
  *
  ****************************************************************************/
 
-void cxx_initialize(void)
+void lib_cxx_initialize(void)
 {
 #ifdef CONFIG_HAVE_CXXINITIALIZE
   static int inited = 0;
 
   if (inited == 0)
     {
+#if defined(CONFIG_ARCH_SIM) && defined(CONFIG_HOST_MACOS)
+      extern void macho_call_saved_init_funcs(void);
```

Comment:
Maybe we can have it as a part of `External References` section?

Reply:
Done.